Resources/Textures/Clothing/Neck/Misc/pins.rsi/meta.json

```
@@ -98,6 +98,10 @@
             "name": "omni-equipped-NECK",
             "directions": 4
         }
+        {
+            "name": "gend-equipped-NECK",
+            "directions": 4
+        }
```

incorrect name, won't be seen on the neck
```suggestion
        {
            "name": "gender-equipped-NECK",
            "directions": 4
        }
```